cloud-functions/index.js

```diff
@@ -15,6 +15,7 @@ app.listen(DEBUG_PORT, () => {
 
 app.get('/posts', postsController.getAllPosts)
 app.get('/posts/:postId', postsController.getPostWithId)
+app.post('/posts', jsonParser, postsController.postPost)
```

Comment:
bu jsonParser olarak gecirdigimiz sey nedir?

Reply:
gelen requesti parse edecek middleware. requesti json olark attığımız için ihtiyaç duyuyoruz